accelerator/bxb/board/ultra96/Makefile

```diff
@@ -0,0 +1,30 @@
+CHISEL_FILES := $(wildcard ../../src/main/scala/**/*.scala)
+
+
+.NOTPARALLEL: all
+
+all:  rtl/Bxb.v vivado vivado_gen
+
+rtl/Bxb.v: $(CHISEL_FILES)
+	cd ../../ && sbt "runMain bxb.Bxb --output-file $(abspath $@)"
+
+.PHONY: vivado
+vivado: 
```

Comment:
Does this one depend on Verilog generation?
Could we add a dependency from `rtl/Bxb.v` file?

Reply:
Updated Makefile.